lib/clients.ex

```diff
@@ -28,7 +28,7 @@ defmodule GlassFactoryApi.Clients do
   """
   @spec get_client(integer(), map) :: {atom(), Client.t() | String.t()}
   def get_client(client_id, config \\ %{}) do
-    with {:ok, %{status: 200, body: body}} <- ApiClient.get("clients/#{client_id}", config) do
+    with {:ok, %{status: 200, body: body}} <- ApiClient.get("clients/#{client_id}", [], config) do
```

Comment:
We shouldn't need to make these changes, because `[]` is already the default value.